### DISCUSSION TITLE
[Potential Bug] Production build error - SyntaxError: The requested module 'react' does not provide an export named 'useEffect'

### DISCUSSION THREAD

**Discussion by MatMorris** [tailwindlabs/headlessui#1460] [Help]
I get this error when trying to build for production a previously working deployment after npm update. Does anyone have any ideas?

`
08:29:23.184 | > Build error occurred
-- | --
08:29:23.186 | file:///opt/buildhome/repo/node_modules/@headlessui/react/dist/hooks/use-disposables.js:1
08:29:23.186 | import{useState as s,useEffect as o}from"react";import{disposables as t}from"../utils/disposables.js";function p(){let[e]=s(t);return o(()=>()=>e.dispose(),[e]),e}export{p as useDisposables};
08:29:23.187 | ^^^^^^^^^
08:29:23.187 | SyntaxError: The requested module 'react' does not provide an export named 'useEffect'
08:29:23.187 | at ModuleJob._instantiate (internal/modules/esm/module_job.js:92:21)
08:29:23.187 | at async ModuleJob.run (internal/modules/esm/module_job.js:107:20)
08:29:23.188 | at async Loader.import (internal/modules/esm/loader.js:179:24) {
08:29:23.188 | type: 'SyntaxError'
08:29:23.189 | }
`

**Comment by fembuelita** [comment]
Side note: I just started getting this same error today after `rm -rf node_modules` and a fresh install using Node 14 on a different project not using this package yesterday. Seems like something recently updated may have caused this
